types/three/examples/jsm/libs/lil-gui.module.min.d.ts

```diff
@@ -324,7 +324,7 @@ export class ColorController<T = Record<string, unknown>, K extends keyof T = ke
  * lil-gui
  * https://lil-gui.georgealways.com
  * @version 0.17.0
- * @author George Michael Brower
+ * @author George Michael Browser
```

Comment:
Are you sure this is change is correct? Looks like Browser in the [lil-gui package.json](https://github.com/georgealways/lil-gui/blob/master/package.json#L5).